server/licenses/ehcache-NOTICE.txt

```diff
@@ -0,0 +1,5 @@
+Ehcache V3
+Copyright 2014-2023 Terracotta, Inc.
```

Comment:
2024 ?

Reply:
I copied it from ehcache library, still says 2023. So keeping this as it is.